### PR TITLE
CFY-2841: improve network_plugin test coverage

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: vcloud_network_plugin/floatingip.py [hunk 1]
```diff
@@ -129,7 +129,7 @@ def _floatingip_operation(operation, vca_client, ctx):
         if SSH_PUBLIC_IP in ctx.source.instance.runtime_properties:
             del ctx.source.instance.runtime_properties[SSH_PUBLIC_IP]
         if SSH_PORT in ctx.target.instance.runtime_properties:
-            del ctx.source.instance.runtime_properties[SSH_PORT]
+            del ctx.target.instance.runtime_properties[SSH_PORT]
```
Comment (kostya13) [inline vcloud_network_plugin/floatingip.py]: Why did you change 'source' to 'target'?

Comment (0lvin) [inline vcloud_network_plugin/floatingip.py]: In code on one line above - checked  ctx.target.instance not ctx.source.instance

Comment (kostya13) [inline vcloud_network_plugin/floatingip.py]: PUBLIC_IP belong to ctx.target instance. (floating ip node)
SSH_PUBLIC_IP and  SSH_PORT belong to ctx.source (server node).
They are created in save_ssh_parameters() function.

**Comment by kostya13** [comment]
Looks good +1
